easybuild/easyconfigs/l/launcher/launcher-3.7-foss-2020a.eb

```diff
@@ -0,0 +1,51 @@
+# Autor:  Felipe Franco <felipe.franco@hpcnow.com>
+#
+easyblock = 'Tarball'
+
+name = 'launcher'
+version = '3.7'
+
+homepage = 'https://github.com/TACC/launcher'
+description = """
+Launcher is a utility for performing simple, data parallel, 
+high throughput computing (HTC) workflows on clusters, 
+massively parallel processor (MPP) systems, workgroups of computers, 
+and personal machines.
+"""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+
+sources = [{
+    'filename': '%(name)s-%(version)s.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/tacc',
+        'repo_name': 'launcher',
+        'tag': 'v3.7',
+        'recursive': True,
+        'keep_git_dir': True,
+    },
+}]
+
+checksums = ['7a3cfc16e4882d9fa08093b1035b1f5e8733477246ca1088cfdc27fc30bf9b90']
+
+dependencies = [
+    ('hwloc', '2.2.0', '', ('GCCcore', '9.3.0')),
```

Comment:
See above - when we are on the `GCCcore/9.3.0` toolchain, we don't need to specify it here.
Please also add a `Python` dependency, given that there are some `Python` scripts in here. Please also check which version of `Python` it needs.

Reply:
Done. Please see the merged PR below